.github/workflows/e2e-node.yml

```diff
@@ -15,11 +15,17 @@ jobs:
     environment:
       name: ${{ matrix.environment-name }}
     strategy:
+      fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        node-version: [16.x]
-        # NSS does not support static client registration, which we rely on for testing.
-        environment-name: ["ESS Dev-Next"]
+        node-version: ["14.x", "16.x"]
+        environment-name: ["ESS PodSpaces", "ESS PodSpaces Next"]
+        experimental: [false]
+        include:
+          - version: "18.x"
+            experimental: true
+          - environment-name: "ESS Dev-Next"
+            experimental: true
```

Comment:
```suggestion
            node-version: ["14.x", "16.x"]
            experimental: true
```

Reply:
Note that this isn't possible in the `include` section of the matrix: https://github.com/inrupt/solid-client-vc-js/actions/runs/3542226441